pennylane/ops/qubit.py

```diff
@@ -480,13 +480,13 @@ class CRot(Operation):
 
 class QubitUnitary(Operation):
     r"""QubitUnitary(U, wires)
-    Apply an arbitrary unitary matrix
+    Apply an arbitrary fixed unitary matrix; QubitUnitary only accepts unitary matrices which do not depend on any parameters.
```

Comment:
I would suggest removing the second sentence, my feeling is that 'fixed unitary' is descriptive enough. The second statement I find a bit confusing, as the unitary _can_ be parametrized, it just can't depend on any parameters that we differentiate with respect to.
```suggestion
    Apply an arbitrary fixed unitary matrix.
```